pom.xml

```diff
@@ -2505,6 +2511,18 @@
                 </dependencies>
             </dependencyManagement>
         </profile>
+        <profile>
+            <id>build-with-jdk-11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <properties>
+                <project.build.targetJdk>11</project.build.targetJdk>
+                <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Personal opinion: I'm not fond of moving past Java 8 for source and binary support at this time. There's a lot more Java 8 out there than most devs realize, and moving past Java 8 opens up a lot of problems with compatibility for very little and even negative benefit. Running in later JDKs is fine, but does not require updating the source and target JDK.

Reply:
A lot of things cannot be solved if we keep Java 8, like lib updates, and CVE fixes, which prevent Presto from broader adoption.
The PR does not require any source code updates, some test code updates are needed, through.
CC @tdcmeehan

Reply:
To hypotheticals and maybes I respond YAGNI! Is there a specific problem that requires Java 11? If so, file an issue for it. I don't personally know of anyone failing to adopt Presto due to lib issues or CVEs. If there is, we need details.

Reply:
here is one requiring jdk11
https://github.com/prestodb/presto/pull/22283